02-express-tutorial/app.js

```diff
@@ -1 +1,166 @@
-console.log('Express Tutorial')
+console.log("Express Tutorial");
+
+const express = require("express");
+const { products } = require("./data");
+const { people } = require("./data");
+const peopleRouter = require("./routes/people");
+const app = express();
+
+app.use(express.static("./public"));
+
+console.log("Static files are being served from ./public");
+
+const logger = (req, res, next) => {
+    console.log(`${req.method} ${req.url} at ${new Date().toISOString()}`);
+    next();
+};
+app.use(logger);
+
+app.use("/api/v1/people", peopleRouter);
```

Comment:
Generally speaking it is better to define all your middleware before mounting any routes (unless you have a specific reason for it) i.e. it would be better to move this line to after line 22:
```javascript
// ... define all your middlewares
app.use(express.json());
// then mount your routers
app.use("/api/v1/people", peopleRouter);
```